.github/scripts/runner.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
i think this is a bad idea since it seems like it'd ignore the python binary within a venv